src/geometry/GeometryEvaluator.cc

```diff
@@ -342,6 +355,8 @@ Polygon2d *GeometryEvaluator::applyToChildren2D(const AbstractNode& node, OpenSC
     return applyMinkowski2D(node);
   } else if (op == OpenSCADOperator::HULL) {
     return applyHull2D(node);
+  } else if (op == OpenSCADOperator::FILL) {
```

Comment:
Am wondering if this section should now be changed to a switch statement to match switch statement a little later in the function?

Reply:
Yep, might be nice, but I also don't mind keeping the if/else.